src/app.js

```diff
@@ -62,6 +62,7 @@ var accountTradeHandler = function (username, password, sharedSecret) {
     });
 
     client.on("loggedOn", function () {
+        this.setPersona(SteamUser.EPersonaState.Online) // we want to appear Online 24/7
```

Comment:
Please don't write comments on the side. Put it to line above and just add blank lines for readability.